src/main/java/core/basesyntax/service/write/FileWriterImpl.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.service.write;
+
+import java.io.BufferedWriter;
```

Comment:
According to the checklist, remember to place the input and output files into the `src/main/resources` folder and use relative paths. Make sure that any file paths used in the application adhere to this guideline.